### DISCUSSION TITLE
to be ignored: INFO: Could not find files for the given pattern(s).

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#972] [General]
Best practice whenever you get spammed in your terminal with:
`INFO: Could not find files for the given pattern(s).`
or in German:
`INFORMATION: Es konnten keine Dateien mit dem angegebenen Muster gefunden werden.`
is to keep calm and ignore it.

This is not brownie specific and when searching for it you will find this message is popping up in all kinds of contexts, not even necessarily python related.

The best info I could find was [here](https://discourse.mc-stan.org/t/unable-to-install-cmdstan-could-not-find-files-for-the-given-pattern-s/23628/4)

> `INFO: Could not find files for the given pattern(s).`
> is not actually that problematic. Its just an informational message that pops up on Windows (some patterns used when building only work for Linux not on Windows), but can be ignored.

Anyway, if anyone finds out how to fix this, it would be much appreciated.

**Comment by cromewar** [comment]
Nice info, also this is something related to the instruction translation between `powershell and cmd` on windows, more info here: https://stackoverflow.com/questions/43494895/executing-powershell-command-under-cmd-and-info-could-not-find-files-for-the-gi

**Reply by n4n0b1t3** [reply]
oh, yes! I ran down this rabbit hole too 👯
